app/actions/utils.js

```diff
@@ -205,24 +205,24 @@ export function mergeProfileUpdates(profile, updates){
 
 export function sendToRollbar(err, props) {
   return new Promise((resolve) => {
-    if (rollbar) {
-      const extra = {};
-      if (_.get(props, 'data.blobId', false)) {
-        _.assign(extra, { blobId: props.data.blobId });
-      }
+    if (!rollbar) {
+      return resolve(props);
+    }
 
-      rollbar.error(err, extra, (err, data) => {
-        if (err) {
-          console.log('Error while reporting error to Rollbar:', err);
+    const extra = {};
+    if (_.get(props, 'data.blobId', false)) {
+      _.assign(extra, { blobId: props.data.blobId });
+    }
+    
+    rollbar.error(err, {...extra, ... props}, (reportingErr, data) => {
```

Comment:
Is there a gap between `...` and `props`? We should probably remove it.

Reply:
Hmm, I see now we have `extra` and `props`. Why not combine them earlier, by doing a `const extra = { ...props }`, instead of having to use two spread operators here.